app/scripts/TiledPixiTrack.js

```diff
@@ -740,11 +747,25 @@ class TiledPixiTrack extends PixiTrack {
         uuid: this.uuid
       });
     }
+    const errors = Object.values(this.fetchedTiles)
+      .map(
+        x =>
+          x.tileData && x.tileData.error && `${x.tileId}: ${x.tileData.error}`
```

Comment:
I am curious why `${x.tileId}: ${x.tileData.error}` is needed. Could you explain?

Reply:
That just shows the error message. So if there's an error in a tile you'll something like `x.0.14: Poorly formed tile`. At the end of the call chain, we'll end up with an array of errors: `['x0.0: Tile too large', 'x.1.1: Tile malformed']` and then those will be displayed to the user.